packages/core/src/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -42,11 +42,17 @@ ReactDOM.render(
       ]}
       label="Select from the options below:"
       onChange={selectedItem => console.log(selectedItem)}
+      onInputValueChange={inputVal =>
+        console.log('[AUTOCOMPLETE_VALUE]: ' + inputVal)
```

Comment:
Maybe just make these logs consistent with what kind of stuff they're printing out

Reply:
Agreed. Standardizing on [Component]: Event Type || Value for strings, and plain objects.